packages/create/src/cli/index.ts

```diff
@@ -0,0 +1,74 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import { Command } from 'commander';
+import { getLogger } from '../tracing';
+import { getAddCommands } from './add';
+import { getRemoveCommands } from './remove';
+
+/*
+ * If chosen 'commander' over 'minimist' and 'yargs' for this CLI implementation. Reasons:
+ * (if it still up: https://npmtrends.com/commander-vs-minimist-vs-yargs)
+ * At the time this was written in August 2022, 'commander' was the most feature rich, active, and popular module with decent size and built-in types support
+ * - 'minimist' is super small but very limited functionality, no automatic help support
+ * - 'yarg' was in closer consideration, but larger (1.7 times) and not as feature rich
+ */
+
+/**
+ * Run the command line interface.
+ *
+ * @param argv - arguments, typically 'process.argv'
+ */
+export function handleCreateFioriCommand(argv: string[]): void {
+    const logger = getLogger();
+    if (!Array.isArray(argv) || argv.length < 2) {
+        throw Error(`This function must be called from command line interface (cli). Or provide meaningful argv.`);
+    }
+    try {
+        const program = getCommanderProgram();
+        program.parse(argv);
+    } catch (error) {
+        logger.error(`Error while executing ${argv[0]} ${argv[1]}`);
+        if (error instanceof Error) {
```

Comment:
Why narrow the instance type to `Error`? It's a pity that any object can be `throw`n in JS/TS but the serialized value may still be useful

Reply:
changed it to check for type `error` or `string`, as this is what the logger can digest.

Reply:
I think the type definitions of the `error` method in `ToolsLogger` is not accurate. We're basically re-exporting Winston's typedefs. 
The only requirement is that the argument to `error()` must be a string or any type that has a `toString()` method (which all types do).
This check is not required:
https://github.com/SAP/open-ux-tools/blob/b54f1b44870338306df7e89a8d4db33021419fb7/packages/create/src/cli/index.ts#L31-L33
Did you see a problem with not having that check before calling `logger.error()`?

Reply:
> Did you see a problem with not having that check before calling `logger.error()`?

Yes:

<img width="905" alt="image" src="https://user-images.githubusercontent.com/66327622/205366487-7aeb2c14-3c00-4f0e-8ac3-32bd4c0b94c6.png">

There are several ways to overcome this, none seems completely clean:

```typescript
} catch (error: any) {
        logger.error(`Error while executing ${argv[0]} ${argv[1]}`);
        logger.error(error);
```

```typescript
    } catch (error) {
        logger.error(`Error while executing ${argv[0]} ${argv[1]}`);
        logger.error(error as object | string);
```

For now I go with `logger.error(error as object | string);`. What do you think?